doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1959,6 +1959,58 @@ Trusted storage implements the PSA Certified Secure Storage APIs without TF-M.
         - Supported
         - Supported
 
+MCUboot bootloader
+******************
+
```

Comment:
Need to add an intro sentence here :

_The following table indicates the software maturity levels of the support for each MCUboot bootloader feature:_